src/elliptic/curves/secp256_r1.rs

```diff
@@ -0,0 +1,972 @@
+use crate::arithmetic::traits::{Converter, Modulo};
+use crate::elliptic::curves::traits::{ECPoint, ECScalar};
+use crate::{BigInt, ErrorKey};
+
+use rand::{thread_rng, Rng};
+use ring::signature::{
+    cpu,
+    ec::{
+        suite_b::{curve::P256, ops::*},
+        PublicKey, Seed,
+    },
+    untrusted,
+};
+use serde::de;
+use serde::de::{MapAccess, SeqAccess, Visitor};
+use serde::ser::SerializeStruct;
+use serde::ser::{Serialize, Serializer};
+use serde::{Deserialize, Deserializer};
+use std::fmt;
+use std::ops::{Add, Mul};
+use std::ptr;
+use std::sync::atomic;
+use zeroize::Zeroize;
+
+/// The size (in bytes) of a message
+pub const MESSAGE_SIZE: usize = 32;
+
+/// The size (in bytes) of a P-256 signature
+pub const SIGNATURE_SIZE: usize = 64;
```

Comment:
what is P-256 signature ? 
where do we use this const ?

Reply:
You're right, we don't we use it.
I'll push a commit to delete it.